Add content for Nodes, Workloads, Dashboard and Security Events and JIRA NVSHAS-7522, NVSHAS-9293 and NVSHAS-8148 (#109)
neuvector/docs: docs/15.ui_extension/02.runtime _security/runtime_security.md

---
title: 5.x Runtime Security
taxonomy:
category: docs
slug: /ui_extension/runtime_security
---

### Assets Security Scan Result

> As Rancher has its own Nodes and Workloads page to manage and monitor the assets' status, NeuVector injects components to display its runtime scan result.

Reviewer: ```suggestion
As Rancher has its own Nodes and Workloads page to manage and monitor the assets statuses, NeuVector injects components to display its runtime scan result.
```

Author: Fixed

#### Nodes

> In nodes summary table, vulnerabilities column is injected to show each node's detected vulnerabilities amount by high / medium severity.

Reviewer: ```suggestion
In the nodes summary table, the vulnerabilities column is injected to show each node's detected vulnerabilities amount by high / medium severity.
```

Author: Fixed

![Nodes](nodes.png)

> Click on the vulnerabilities cell, a slide panel comes from right side of the page to show scan result details. The database version of the scanner is shown as well as a filterable vulnerability list.

Reviewer: ```suggestion
Click on the vulnerabilities cell, a slide panel will come from the right side of the page to show scan result details. The database version of the scanner is shown as well as a filterable vulnerability list.
```

Author: Fixed

![Nodes](nodes_vul.png)

> Click on the node name cell on nodes table, it redirects to node detail page. NeuVector inject a tab to show vulnerability list.

Reviewer: ```suggestion
Click on the node name cell on nodes table, it redirects to the node detail page. NeuVector injects a tab to show the vulnerability list.
```

Author: Fixed

![Nodes](node_detail_vul.png)

Click on the CVE name cell on the vulnerabilities table, it opens a popup to show CVE description with an external link on the header to SUSE's CVE information website.

![Nodes](node_detail_vul_cve.png)

![SUSE](suse_cve.png)

> A CSV file downloading button is provided above the vulnerabilities table. The file data includes all the fields which are in the table.

Reviewer: ```suggestion
A CSV file download button is provided above the vulnerabilities table. The file data includes all the fields which are in the table.
```

Author: Fixed

![CSV](download_vul.png)

![CSV](cve_csv.png)

#### Workloads

> Temporarily, vulnerabilities column will not be injected in workloads table in v5.4 release due to potential performance concern. NeuVector team will work with Rancher team to deal with it and integrate it soon.

Reviewer: ```suggestion
Temporarily, the vulnerabilities column will not be injected in the workloads table in the v5.4 release due to potential performance concern. The NeuVector team will work with the Rancher team to integrate it soon.
```

Author: Fixed

![Workload](pods.png)

> Click on a pod name on the workloads table, it redirects to pod detail page. NeuVector inject a tab to show vulnerability list.

Reviewer: ```suggestion
Click on a pod name in the workloads table, it redirects to the pod detail page. NeuVector injects a tab to show the vulnerability list.
```

Author: Fixed
